src/styles/components/_component-card.scss

```diff
@@ -73,7 +73,10 @@
   }
 
   &:focus .component-card {
-    outline: none;
-    border: 2px solid var(--accent-color);
+    position: relative;
+    z-index: 1;
+    outline: 2px solid var(--accent-color);
+    outline-offset: 2px;
+    transform: rotate(2deg);
```

Comment:
Going back and looking again this morning, I realized it looks great in light mode but still looks a little funny in dark mode:

![image](https://user-images.githubusercontent.com/54716846/206735242-0d8ed699-c5fa-4b94-9edb-910fb3118fce.png)


I played around with it a bit in the browser and I think a good solution would be:
- add a white background-color to the `.component-card` on focus for dark mode
- add a black background-color to the `.component-link` on focus for dark mode
- don't forget to scope changes so that these colors are only applied for focus state and in dark mode!
- I'm guessing there might be variable color names (like `$white` or `$primary` or maybe `var(--black)`, use whatever those are instead of `white`/`black`

```suggestion
    transform: rotate(2deg);
    
    @media screen and (prefers-color-scheme: dark) {
       background-color: white;
     }
```